test/test_meta.py

```diff
@@ -940,4 +944,6 @@ def print_op_str_if_not_supported(op_str):
                 print_op_str_if_not_supported(op_str.strip())
         sys.exit(0)
 
-    run_tests()
+    import torch._dispatch.python as d
+    with d.enable_python_dispatcher():
+        run_tests()
```

Comment:
This should be obsoleted by https://github.com/pytorch/pytorch/pull/86228 cc @bdhirsh